frontend/src/assets/css/variables.scss

```
@@ -12,4 +12,5 @@
   --blue: #3b79c3;
   --text-type-400: #130138;
   --text-type-300: #5a6474;
+  --z-index-modal: 10;
```

As not it's global variable - then we don't need to name it as related to modal
```suggestion
  --z-index-level-one: 10;
```